### DISCUSSION TITLE
Gruppenzusammensetzung Gruppe 3: Gruppe 1

### DISCUSSION THREAD

**Discussion by eacikgoe** [irgroup-classrooms/dis08-2021#53] [Q&A]
Hallo Herr Schaer und Herr Haak,

wir haben gestern hier unsere Gruppe mit dem Namen Gruppe 3: Gruppe 1 erstellt. 2 der Gruppenteilnehmer sind schon in der Gruppe, jedoch beide doppelt aufgelistet, sodass die Gruppe voll ist. Jedoch haben wir nun 2 weitere Gruppenteilnehmer, die sich jetzt nicht in die Gruppe einschreiben können. Betroffen sind eacikgoe und lucyschretter. Können Sie uns da weiterhelfen?

Viele Grüße
Emirhan Acikgöz

**Comment by phschaer** [comment]
Kann ich - Habe die beiden fehlenden Personen hinzugefügt. 

Aber mal im Vertrauen... Über die Namensgebung reden wir beizeiten nochmal... :-)

**Reply by eacikgoe** [reply]
Super, danke Ihnen :)
